css/modals/invite/_info.scss

```diff
@@ -56,11 +56,11 @@
     }
 
     .info-dialog-conference-url {
-        max-width: 250px;
-        overflow: hidden;
-        text-overflow: ellipsis;
-        user-select: text;
-        white-space: nowrap;
+        width: max-content;
+        width: -moz-max-content;
+        width: -webkit-max-content;
```

Comment:
Drive by comment: do you have any thoughts on Autoprefixer, to avoid writing the prefixed versions ourselves? https://github.com/postcss/autoprefixer

Reply:
That's a good idea. I made a ticket for myself to add it. I'll try to get to it within the next month. Should be quick.

Reply:
Awesome!